cunumeric/_ufunc/ufunc.py

```diff
@@ -211,7 +221,14 @@ def _maybe_cast_input(self, arr, to_dtype, casting):
 
         return arr.astype(to_dtype)
 
-    def _maybe_create_result(self, out, out_shape, res_dtype, casting, inputs):
+    def _maybe_create_result(
+        self,
+        out: Union[ndarray, None],
+        out_shape: tuple[int, ...],
+        res_dtype: np.dtype[Any],
+        casting: CastingKind,
+        inputs: Union[tuple[ndarray, bool], tuple[ndarray, ndarray, bool]],
```

Comment:
OK, I will make `create_auto_task` and `create_manual_task` that delegate to `create_task` but since that will require `legate` changes I propose to merge this for now, and then make update in the next mypy PR once the legate change is also merged.